### PR TITLE
docs: update pre-commit hook

### PR DESCRIPTION
**Pull request by efiop**
https://discordapp.com/channels/485586884165107732/563406153334128681/762305272512249856

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by shcheklein** [comment]
@rogermparent could you check please what's going on with the link check?

File: content/docs/command-reference/install.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ repos:
         stages:
           - commit
       - id: dvc-pre-push
+        additional_dependencies: ['.[all]']  # use s3/gs/etc instead of all to only install specific cloud support
```
Comment (shcheklein) [inline content/docs/command-reference/install.md:92]: can we make it fit the screen? move to the line up and split?

**Comment by rogermparent** [comment]
> @rogermparent could you check please what's going on with the link check?

I messed something up with the `merge-base` patch apparently, let me check it out.

**Comment by shcheklein** [comment]
@rogermparent thanks for the update!!

**Review by shcheklein — APPROVED** [review]

**Pull request merged**